giftexpertapp/src/components/GifGrid.jsx

```diff
@@ -0,0 +1,52 @@
+import axios from "axios"
+import { GIPHYAPI } from "../utils/strings"
+import { useEffect, useState } from "react"
+import GifItem from "./GifItem"
+
+const getGifs = async (category) => {
+    const url = `https://api.giphy.com/v1/gifs/search?api_key=${GIPHYAPI}&q=${category}&limit=20`
+    const response = await axios.get(url)
+    const gifs = response.data.data.map((gif) => {
+        return {
+            id: gif.id,
+            url: gif.images.original.url,
+            title: gif.title
+        }
+    })
+
+    return gifs
+}
```

Comment:
Apis should be present in a separate layer. Do not write api related code in screens